react/components/CustumerInfo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { profileShape } from '../shapes'
+import { intlShape, injectIntl } from 'react-intl'
+import { ProfileRules, ProfileSummary } from 'vtex.profile-form'
+
+const CustumerInfo = ({ profile, intl }) => (
+  <div className="flex flex-column c-on-base">
+    <ProfileRules country={intl.locale} shouldUseIOFetching>
+      <ProfileSummary profile={profile}>
+        {({ personalData }) => (
+          <ul className="list pl0 c-on-base">
```

Comment:
Since you already put the `c-on-base` on the parent div I don't think that you need to repeat it here

Reply:
Yep. Removed unnecessary `c-on-base`.